code/lib/core-server/src/presets/common-preset.ts

```diff
@@ -232,3 +240,60 @@ export const managerHead = async (_: any, options: Options) => {
 
   return '';
 };
+
+const WHATS_NEW_CACHE = 'whats-new-cache';
+const WHATS_NEW_URL = 'https://storybook-dx.netlify.app/.netlify/functions/whats-new';
+
```

Comment:
@winkerVSbecks Could we make an URL in storybook.js.org for this? @ndelangen pointed out that this URL will keep living in SB 7.1 for eternity, so we probably want something where we have more control about and not being tightly coupled to netlify.

Reply:
@winkerVSbecks We might also want to postfix it with `v1` so that we have control over making a `v2` endpoint in the future, without doing a breaking change to old clients.

Reply:
Yup, I can route this through storybook.js.org frontpage and add the v1 suffix

Reply:
The new url is `https://storybook.js.org/v1/whats-new`. 
We often publish a preview of the post. You can access the preview post using `https://storybook.js.org/v1/whats-new?preview=true`. This will also be helpful to the test the feature since we don't have a what's new post yet, just one in preview.

Reply:
@winkerVSbecks Could we do?
`https://storybook.js.org/whats-new/v1`

I think that would make more sense. To version the endpoint independent of other endpoints.